contrib_bots/lib/googlesearch.py

```diff
@@ -0,0 +1,78 @@
+# See zulip/contrib_bots/lib/readme.md for instructions on running this code.
+from google import search
+import logging
+
+def get_google_result(original_content):
+    original_content = original_content.split(' ', 1)[1]
+    if original_content == '--help':
+        help_message = '''
+                        To use this bot start message with @google followed
+                        by what you want to search for. If found, Zulip will
+                        return the first search result on google.
+                        '''
+        return help_message
+    else:
+        try:
+            urls = search(original_content, stop=20)
+        except:
+            logging.exception('no internet connection')
+            return "Error: search failed."
+
+        if not urls:
+            return "No URLs returned by google."
+
+        url = next(urls)
+
+        return 'Success: ' + str(url)
+
+class GoogleSearchHandler(object):
+    '''
+    This plugin allows users to enter a search
+    term in Zulip and get the top URL sent back
+    to the stream in which it was called.It looks
+    for messages starting with @google.
+    '''
+
+    def usage(self):
+        return '''
+            This plugin will allow users to search
+            for a given search term on Google from
+            Zulip. If mor help is needed with using
+            this bot, use '@google --help'. Users
+            should preface messages with @google.
+            '''
+
+    def triage_message(self, message):
```

Comment:
The bot api has changed, you need to update your argument list. Have a look at the stream `GCI Announce/contrib_bot Update`.